### PR TITLE
Done the changes and CRUD too i think.

### PR DESCRIPTION
**Pull request by hamzafj5**

### CODE REVIEW

File: CustomeDB/CustomeDB.cpp [hunk 1]
```diff
@@ -2,10 +2,15 @@
 #include <string>
 #include "LinkedList.h"
 #include "Test.h"
+#include "Activity.h"
 using namespace std;
 
 int main()
 {
+    Activity test;
```
Comment (haseebuchiha) [inline CustomeDB/CustomeDB.cpp]: Don't change anything in main . Hint Unstage main before push

**Review by haseebuchiha — CHANGES_REQUESTED** [review]

**Review by hamzafj5 — COMMENTED** [review]
Done the changes and CRUD too i think

File: CustomeDB/Activity.cpp [hunk 2]
```diff
@@ -1,5 +1,28 @@
...
+
+void Activity::setdata(string name, int cl_id, string phone, string address, int project_id) {
+	cli.setname(name);
+	cli.setcl_id(cl_id);
+	cli.setphone(phone);
+	cli.setaddress(address);
+	cli.setproject_id(project_id);
+}
+
+void Activity::entertoclidatabase() {
```
Comment (ParanoidBat) [inline CustomeDB/Activity.cpp:12]: follow camelCase naming convention for functions

File: CustomeDB/Activity.h [hunk 3]
```diff
@@ -4,14 +4,22 @@
...
-	void enterToDatabase();
-	void displayDatabase();
+	void setdata(string name, int cl_id, string phone, string address, int project_id);
+  
+  void enterToDatabase();
+  void entertoclidatabase();
+	
+  void updateprojectidclidatabase(int project_id, int new_project_id);
+	
+  void displayDatabase();
```
Comment (ParanoidBat) [inline CustomeDB/Activity.h:21]: create meaningful names for functions. this function and one below it have same names!

File: CustomeDB/LinkedList.cpp [hunk 4]
```diff
@@ -54,21 +96,21 @@ Node* LinkedList::getclientHead()
...
+    else {
+        Node* temp = emphead;
+        emphead = new_node;
+        emphead->next = temp;
+    }
 }
 
-void LinkedList::addAtHead(string name, int id, string phone, string address, int project_id) {
-    Node* new_node = new Node(name, id, phone, address, project_id);
+void LinkedList::addAtHead(string name, int cl_id, string phone, string address, int project_id) {
```
Comment (ParanoidBat) [inline CustomeDB/LinkedList.cpp:112]: do not create overloaded functions. just name them differently. there are other such inconsistencies, resolve them all

**Pull request closed without merging**